openshift/kustomize/tekton/base/tasks/deploy-all.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
It will deploy all images that were built.  In addition it stops all services before deploying the API.